letsencrypt/cli.py

```diff
@@ -878,9 +963,11 @@ def prepare_and_parse_args(plugins, args):
                 "multiple -d flags or enter a comma separated list of domains "
                 "as a parameter.")
     helpful.add(
-        None, "--duplicate", dest="duplicate", action="store_true",
-        help="Allow getting a certificate that duplicates an existing one")
-
+        None, "--keep-until-expiring", "--keep", "--reinstall",
```

Comment:
What is the benefit of these aliases? I personally feel that it complicates the command line with little to no gain.